tox.ini

```diff
@@ -29,6 +29,13 @@ exclude =
   tests/packages,
   semver.py,
 max-line-length=100
+# D100  Missing docstring in public module
+# D101  Missing docstring in public class
+# D102  Missing docstring in public method
+# D103  Missing docstring in public function
+# D104  Missing docstring in public package
+# D105  Missing docstring in magic method
+ignore = D100, D101, D102, D103, D104, D105
```

Comment:
Flake8, by default, doesn't check docstrings. You must have a flake8 plugin installed in your system/environment.

Also refer to the list of default errors: http://flake8.pycqa.org/en/latest/user/error-codes.html